x/incentives/simulation/params.go

```diff
@@ -17,10 +17,10 @@ const (
 
 // TODO: remove hardcoded params
 // refer x/epochs/simulation/genesis.go.
+// TODO: x/epochs/simulation/genesis.go doesn't exist, what was this comment trying to say?
```

Comment:
TODO: x/epochs/simulation/genesis.go doesn't exist, what was this comment trying to say?